### PR TITLE
TF-1194 [Part III] Apply new delete identity design

### PR DESCRIPTION
**Pull request by sherlockvn**
#1194

https://user-images.githubusercontent.com/43041967/215708824-a1c11d84-acf5-4201-96a8-e2baef63c7ca.mp4

https://user-images.githubusercontent.com/43041967/215389744-618c23ff-3632-4d75-bad8-793c2a4ce1c3.mp4

https://user-images.githubusercontent.com/43041967/215556812-12517376-e3f7-4812-b5a4-95c1bb6ddcc8.mp4

### CODE REVIEW

File: lib/features/manage_account/presentation/profiles/identities/widgets/delete_identity_dialog_builder.dart [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+  }) : super(key: key);
+
+  @override
+  Widget build(BuildContext context) {
+    return ResponsiveWidget(
+      responsiveUtils: responsiveUtils, 
+      mobile: (_buildDeleteDialog(context)
+          ..aligment(Alignment.bottomCenter)
+          ..outsideDialogPadding(const EdgeInsets.only(left: 0, right: 0, bottom: 42.0))
+          ..widthDialog(MediaQuery.of(context).size.width * 0.95)
```
Comment (dab246) [inline lib/features/manage_account/presentation/profiles/identities/widgets/delete_identity_dialog_builder.dart:29]: Set `maxWidth=MediaQuery.of(context).size.width - 16`

File: lib/features/manage_account/presentation/profiles/identities/widgets/delete_identity_dialog_builder.dart [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+    required this.onDeleteIdentityAction
+  }) : super(key: key);
+
+  @override
+  Widget build(BuildContext context) {
+    return ResponsiveWidget(
+      responsiveUtils: responsiveUtils, 
+      mobile: (_buildDeleteDialog(context)
+          ..aligment(Alignment.bottomCenter)
+          ..outsideDialogPadding(const EdgeInsets.only(left: 0, right: 0, bottom: 42.0))
```
Comment (dab246) [inline lib/features/manage_account/presentation/profiles/identities/widgets/delete_identity_dialog_builder.dart:28]: Padding `bottom=16`

**Comment by hoangdat** [comment]
Hi, I checked the demo videos, but all the demos still have error in the delete action. What problem? Why the list is not updated after each action?

**Review by nqhhdev — APPROVED** [review]

**Review by dab246 — APPROVED** [review]
lgtm

**Review by nqhhdev — APPROVED** [review]

**Pull request merged**